### ISSUE TITLE
MinOffsetToPull for compact height

### ISSUE THREAD

**Issue by oliverdowling** [gontovnik/DGElasticPullToRefresh#3]
Smaller devices (i.e. iPhones, especially 5s) in landscape barely give enough height to pull to refresh.

I changed the MinOffsetToPull constant to 50 to be suitable for all devices, but it might be preferable to have it dynamic.

What are your thoughts on this value?

**Comment by gontovnik** [comment]
Definitely, I will change it to be a variable in a pull to refresh class so people can change that. :+1:
